src/Plugin/audio_embed_field/Provider/StanfordSoundCloud.php

```diff
@@ -89,7 +88,8 @@ protected static function getVideoData($video_url) {
     }
 
     try {
-      $client = new Client();
+      /** @var \GuzzleHttp\Client $client */
```

Comment:
Did you mean to leave this comment in here?

Reply:
probably dont need it.